scripts/build.ps1

```diff
@@ -119,7 +119,7 @@ $language = @("cs", "de", "es", "fr", "it", "ja", "ko", "pl", "pt-BR", "ru", "tr
 # Capture error state in any step globally to modify return code
 $Script:ScriptFailed = $false
 
-Import-Module "$($CurrentScriptDir.FullName)\verify-nupkgs.ps1"
+Import-Module -Name "$($CurrentScriptDir.FullName)\verify-nupkgs.ps1" -Scope Local
```

Comment:
why? the other file is not even a module (not your fault).  But why local scope?

Reply:
It's a workaround. Sometimes if the `build.ps1` fails or interrupted by `CTRL+C`, subsequent run fails to import `Verify-Nuget-Packages` method this fixes that. I don't know why.